p2p/ipld/plugin/nodes/nodes.go

```diff
@@ -24,6 +25,10 @@ const (
 
 	// Nmt is the codec used for leaf and inner nodes of an Namespaced Merkle Tree.
 	Nmt = 0x7700
+
+	// NmtCodecName is the name used during registry of the Nmt codec
+	NmtCodecName = "nmt-inner-or-leaf-node"
```

Comment:
I think "nmt-node" is more more concise and conveys the same information. As an alternative we could simply use `"nmt"` as each node could also be seen as a (sub-tree).

```suggestion
	NmtCodecName = "nmt-node"
```